doc/nrf/ug_thread_ot_memory.rst

```diff
@@ -77,25 +76,44 @@ The following tables present memory requirements for samples running on the :ref
-   | :ref:`CLI <ot_cli_sample_minimal>` | mtd       |                               298 |                            74 |                        24 |             116 |                                79 |                            11 |              38 |
-   +------------------------------------+-----------+-----------------------------------+-------------------------------+---------------------------+-----------------+-----------------------------------+-------------------------------+-----------------+
+   +------------------------------------+-----------+---------------------------+----------------------+---------------------------+-----------------+---------------------------+----------------------+-----------------+
+   | Sample                             | Variant   |   ROM OT stack + App [kB] |   ROM BLE stack [kB] |   Persistent storage [kB] |   Free ROM [kB] |   RAM OT stack + App [kB] |   RAM BLE stack [kB] |   Free RAM [kB] |
+   +====================================+===========+===========================+======================+===========================+=================+===========================+======================+=================+
+   | :ref:`CLI <ot_cli_sample_minimal>` | master    |                       383 |                   63 |                        24 |              42 |                        89 |                   10 |              29 |
+   +------------------------------------+-----------+---------------------------+----------------------+---------------------------+-----------------+---------------------------+----------------------+-----------------+
+   | :ref:`CLI <ot_cli_sample_minimal>` | ftd       |                       365 |                   63 |                        24 |              60 |                        88 |                   10 |              30 |
+   +------------------------------------+-----------+---------------------------+----------------------+---------------------------+-----------------+---------------------------+----------------------+-----------------+
+   | :ref:`CLI <ot_cli_sample_minimal>` | mtd       |                       313 |                   63 |                        24 |             112 |                        80 |                   10 |              38 |
+   +------------------------------------+-----------+---------------------------+----------------------+---------------------------+-----------------+---------------------------+----------------------+-----------------+
+
+.. _thread_ot_memory_5340:
+
+nRF5340 DK RAM and flash memory requirements
+*********************************************
+
+The following tables present memory requirements for samples running on the :ref:`nRF5340 DK <gs_programming_board_names>` (:ref:`nrf5340dk_nrf5340 <zephyr:nrf5340dk_nrf5340>`) with the hardware cryptography support provided by the CC312.
```

Comment:
Let's add a link to CC312:
```suggestion
The following tables present memory requirements for samples running on the :ref:`nRF5340 DK <gs_programming_board_names>` (:ref:`nrf5340dk_nrf5340 <zephyr:nrf5340dk_nrf5340>`) with the hardware cryptography support provided by the :ref:`CC312 <nrfxlib:nrf_cc3xx_platform_readme>`.
```
If https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/crypto/doc/nrf_cc310_platform.html is the correct link?

Reply:
It should be. Shall we add a link for CC310 above as well?

Reply:
Yes, please - should be the same reference, just different text.